src/onboarding/ha-onboarding.ts

```diff
@@ -127,6 +144,28 @@ class HaOnboarding extends litLocalizeLiteMixin(HassElement) {
     return this._steps ? this._steps.find((stp) => !stp.done) : undefined;
   }
 
+  private _restoringSnapshot() {
+    this._restoring = true;
+  }
+
+  private async _fetchDiscoveryInformation(): Promise<void> {
+    try {
+      const response = await fetchDiscoveryInformation();
+      this._supervisor = [
+        "Home Assistant OS",
+        "Home Assistant Supervised",
+      ].includes(response.installation_type);
+      if (this._supervisor) {
+        // Only load if we have supervisor
+        import(
+          /* webpackChunkName: "onboarding-restore-snapshot" */ "./onboarding-restore-snapshot"
+        );
+      }
+    } catch (err) {
+      alert("Something went wrong loading onboarding, try refreshing");
```

Comment:
We can just log this? Only thing that would not work is restoring a snapshot?

Reply:
True